server/models/notifications/createNotification.js

```diff
@@ -0,0 +1,70 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {Object} param0.body
+ * @return {Promise}
+ *
+ */
+
+function createNotification({ username, body }) {
+  return new Promise((resolve, reject) => {
+    const {
+      heading,
+      description,
+      public,
+      target_usernames: targetUsernames,
+    } = body
```

Comment:
for description, we can use:
```
const description = body.description || null
```

Reply:
Ok, I am on it.

Reply:
I have made changed in schema, that should be sufficient. User can now create a notification with empty description.